src-electron/importexport/import-json.js

```diff
@@ -378,7 +378,7 @@ async function importPackages(
  * @param {*} endpoints
  * @returns list or sorted endpoints based on endpoint type index
  */
-function sortEndpoints(endpoints) {
+async function sortEndpoints(endpoints) {
```

Comment:
I don't get it.  There is nothing async in this function.  Why is it being made async?  This feels like it's covering up a bug somewhere else....

Reply:
agree